sdk/core/README.md

```diff
@@ -1,6 +1,6 @@
 # Azure Core shared library for Java
 
-[![Build Documentation](https://img.shields.io/badge/documentation-published-blue.svg)](https://azure.github.io/azure-sdk-for-java/track2reports/index.html)
+[![Build Documentation](https://img.shields.io/badge/documentation-published-blue.svg)](https://azure.github.io/azure-sdk-for-java)
```

Comment:
I was under the impression this link would be `https://azure.github.io/azure-sdk-for-java/core.html` although it doesn't look like that exists yet. What you do have seems to be a direct replacement so I'm good  with that for now but otherwise I do think we should try to be more specific to the service.

cc @scbedd

Reply:
For right now we're going to leave this linking to https://azure.github.io/azure-sdk-for-java however ALL of the readme's are going to need to be changed to link to their respective areas (core to core, keyvault to keyvault etc)